nifi-nar-bundles/nifi-standard-bundle/nifi-standard-processors/src/main/java/org/apache/nifi/processors/standard/ExecuteStreamCommand.java

```diff
@@ -222,6 +231,16 @@ public ValidationResult validate(String subject, String input, ValidationContext
             .defaultValue(";")
             .build();
 
+    static final PropertyDescriptor PENALIZE_NONZERO_STATUS = new PropertyDescriptor.Builder()
```

Comment:
I wonder if this is necessary? The processor doesn't currently penalize anything. I think it may make sense to always penalize when sending to the new relationship and the user has the ability to just set Penalty Duration to "0 sec" if they don't want to penalize.